### PR TITLE
Better htmlviewer UI

### LINKED ISSUE #12: Better HTMLviewer UI
The [HTMLviewer](http://c0derunr.herokuapp.com/htmlviewer) currently doesn't quite match with the rest of the [c0derunR](http://c0derunr.herokuapp.com/) UI. It would be nice if it could also be taken care of.

### PR DESCRIPTION
**Pull request by 142ayushkumar**
Resolves #12

### CODE REVIEW

File: coyote/views.py [hunk 1]
```diff
@@ -119,4 +119,5 @@ def codeplay(request):
     """
     
     return render(request, 'codeplay.html')
-    
\ No newline at end of file
+
```
Comment (Demfier) [inline coyote/views.py:122]: Remove these extra blank lines. Rest all looks good.

**Comment by kaustubhhiware** [comment]
@Demfier if you happen to merge this, use "squash and merge".
image

**Review by kaustubhhiware — APPROVED** [review]
Squash and merge ready. @Demfier do the honours.

**Comment by Demfier** [comment]
@142ayushkumar - can you please quickly resolve the conflict. It's an easy one.

**Comment by Demfier** [comment]
@142ayushkumar - Another conflict. Pull before you push into the branch.

**Comment by 142ayushkumar** [comment]
Finally, no conflicts. Sorry for the trouble

**Review by Demfier — APPROVED** [review]

**Comment by Demfier** [comment]
@kaustubhhiware - Can you look at the deployment. Is it supposed to happen?

**Comment by kaustubhhiware** [comment]
@Demfier python webapps are hosted on http, not https.
Just change the url.

**Pull request merged**